source/reference/read-concern.txt

```diff
@@ -52,9 +52,9 @@ The following read concern levels are available:
    * - .. readconcern:: "linearizable"
 
      - The query returns data that reflects all successful writes
-       issued with a :writeconcern:`"majority"` *and* acknowledged
+       issued with a write concern of :writeconcern:`"majority"` *and* acknowledged
        prior to the start of the read operation. Linearizable read
-       concern returns data that cannot be rolled back. You can specify
+       concern returns data that will never be rolled back. You can specify
```

Comment:
Where do we say w:majority write concern writes could roll back? I'll check everything for continuity.